addon/-private/system/store.js

```diff
@@ -993,15 +999,14 @@ Store = Service.extend({
     return this._internalModelForId(modelName, id).getRecord();
   },
 
-  _internalModelForId(typeName, inputId) {
+  _internalModelForId(modelName, inputId) {
     heimdall.increment(_internalModelForId);
-    var typeClass = this.modelFor(typeName);
-    var id = coerceId(inputId);
-    var idToRecord = this.typeMapFor(typeClass).idToRecord;
-    var record = idToRecord[id];
+    let id = coerceId(inputId);
+    let schema = this.schemaFor(modelName);
+    let record = schema.recordMap.idToRecord[id];
```

Comment:
this should be internalModel, I was carrying over the old names and should have noticed the old one was bad :trollface: